301-jenkins-aptly-spinnaker-vmss/azuredeploy.json

```diff
@@ -55,8 +61,7 @@
     "publicIPAddressName": "[concat(variables('scenarioPrefix'), 'PublicIP')]",
     "vmName": "[concat(variables('scenarioPrefix'), 'VM')]",
     "vmExtensionName": "[concat(variables('scenarioPrefix'), 'Init')]",
-    "spinnakerOSDiskName": "[concat(variables('scenarioPrefix'), 'OSDisk')]",
-    "_artifactsLocation": "https://raw.githubusercontent.com/Azure/azure-devops-utils/v0.24.0/",
+    "_artifactsLocation": "https://raw.githubusercontent.com/Azure/azure-devops-utils/master/",
```

Comment:
See [BP guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#deployment-artifacts-nested-templates-scripts) for how to handle artifacts - probably need to refactor this a bit to comply...

Reply:
@bmoore-msft , thank you for your review! Updated content for most of your comments. While as for this one that suggests to follow the BP guide, I've initiated an offline discussion with you. Based on that would you please share some best practices?